### PR TITLE
Add reviewer role in Community membership

### PR DESCRIPTION
**Pull request by fisherxu**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea/*
```
Comment (chendave) [inline .gitignore:1]: what's this for?

File: community-membership.md [hunk 2]
```diff
@@ -40,6 +44,28 @@ reviewing issues/PRs or participate in community discussions on slack/mailing li
...
+They are knowledgeable about both the codebase and software engineering principles.
+
+
+### Requirements
+
+- member for at least 1 months
+- Primary reviewer for at least 5 PRs to the codebase
+- Reviewed or merged at least 20 substantial PRs to the codebase
+- Knowledgeable about the codebase
+- May either self-nominate, be nominated by an approver
```
Comment (chendave) [inline community-membership.md:59]:
```suggestion
- May either self-nominate,  or be nominated by an approver
```

File: community-membership.md [hunk 3]
```diff
@@ -40,6 +44,28 @@ reviewing issues/PRs or participate in community discussions on slack/mailing li
...
+
+
+### Responsibilities and privileges
+
+- Code reviewer status may be a precondition to accepting large code contributions
+- Responsible for project quality control
+- Focus on code quality and correctness, including testing and factoring
```
Comment (chendave) [inline community-membership.md:66]: refactoring?  not sure whether this is a typo or not

File: .gitignore (hunk 1 above)
Comment (fisherxu) [reply .gitignore:1]: Just for opening it with goland..

File: community-membership.md (hunk 3 above)
Comment (fisherxu) [reply community-membership.md]: Done.

File: .gitignore (hunk 1 above)
Comment (chendave) [reply .gitignore:1]: not clear though, why the codebase keep something for a specific IDE?

Comment (fisherxu) [reply .gitignore:1]: Goland is widely used in go projects, for go code or some docs.. like K8s: https://github.com/kubernetes/kubernetes/blob/master/.gitignore#L13, 
 Istio https://github.com/istio/istio/blob/master/.gitignore#L10

**Comment by chendave** [comment]
> Goland is widely used in go projects, for go code or some docs.. like K8s: https://github.com/kubernetes/kubernetes/blob/master/.gitignore#L13,
Istio https://github.com/istio/istio/blob/master/.gitignore#L10

I still don't think that is a good behavior to maintain such file in the upstream, besides, I'd suggest to add that in another PR if you do think this is necessary.

**Comment by kevin-wangzefeng** [comment]
manually adding approved label as owners file not checked in

**Pull request merged**